src/qrl/core/BlockHeader.py

```diff
@@ -64,28 +83,53 @@ def tx_merkle_root(self):
 
     @property
     def extra_nonce(self):
+        """
+        If one has run through all possible values of mining_nonce and still not
+        found a hash that satisfies the difficulty, one can use extra_nonce.
+        :return:
+        """
         return self._data.extra_nonce
 
     @property
     def mining_nonce(self):
+        """
+        A meaningless number that the miner can change in order to get different
```

Comment:
Please change the sentence, with better words. Its better to talk about the purpose of the mining nonce.